File: send_msg.php
<?php

// Recebendo parametros
$id_chat = $_POST['chat'];
$mensagem = $_POST['mensagem'];

// Credenciais do telegram
$apiToken = "YOUR KEY";
$data = [
    'chat_id' => $id_chat, 
    'text' => $mensagem
];


// Enviando mensagem
$response = file_get_contents("https://api.telegram.org/bot".$apiToken."/sendMessage?" .
http_build_query($data) );

?>

File: send_msg.py
# Biblioteca para fazer requisões via http
import requests


# Função para enviar mensagens via telagram com pyhton
def send_to_telegram(message):

    # Credencias do meu telegram
    apiToken = 'YOUR KEY'    # Api do meu bot                     
    apiURL = f'https://api.telegram.org/bot{apiToken}/sendMessage' # Api resposanvel por enviar a mensagem
    chatID = '-761330787' # -763496262'   

    try:
        response = requests.post(apiURL, json={'chat_id': chatID, 'text': message})
        print(response.text)
    except Exception as e:
        print(e)


# Chamando função com a mensagem
send_to_telegram("Atenção! algo deu errado... testando envio de mensagem com Python")
